### DISCUSSION TITLE
Try to build the android apk from the source

### DISCUSSION THREAD

**Discussion by shoulih** [cagnulein/qdomyos-zwift#254] [General]
Cloned the latest source from github, opened the android project in Android Studio 4.1.3 and got the following:

Build file 'C:\Users\shu\StudioProjects\qdomyos-zwift\src\android\build.gradle' line: 37

A problem occurred evaluating root project 'android'.
> Could not get unknown property 'androidCompileSdkVersion' for object of type com.android.build.gradle.internal.dsl.BaseAppModuleExtension.

What did I miss?

Thanks,

**Comment by cagnulein** [comment]
You have to use qt creator not Android studio

Il lun 5 apr 2021, 01:40 shoulih ***@***.***> ha scritto:

**Comment by shoulih** [comment]
switched to QT Creator 4.14.2 based on Qt 5.15.2, but got the following now:

error: Project ERROR: Unknown module(s) in QT: networkauth charts

Did I miss any dependency on my Windows 10 64 bit machine?

Thanks,

**Comment by cagnulein** [comment]
Yes, you can find them in the Qt maintenance tool

Il lun 5 apr 2021, 22:36 shoulih ***@***.***> ha scritto:

**Comment by shoulih** [comment]
I have added the following from Qt maintenance tool, but still got ":-1: error: Project ERROR: Unknown module(s) in QT: networkauth charts"

![image](https://user-images.githubusercontent.com/66762397/113740504-98b24900-96b5-11eb-8a7a-a2edc389fa3a.png)

thanks,

**Comment by shoulih** [comment]
ok, qmake is completed w/o any error now

**Comment by cagnulein** [comment]
You added them for the wrong qt version. You showed them for the 5.15.x

Il mar 6 apr 2021, 17:55 shoulih ***@***.***> ha scritto:

**Comment by shoulih** [comment]
got it going finally!  was able to build 'qdomyos-zwift.exe" w/o error; how do I configure the Qt to build the android apk ?

Thanks again,

**Comment by shoulih** [comment]
![image](https://user-images.githubusercontent.com/66762397/113762767-cc00d200-96cd-11eb-878f-0ba576a2a10a.png)

**Comment by cagnulein** [comment]
You need to install the compiler in the maintenance tool for android

Il mar 6 apr 2021, 20:47 shoulih ***@***.***> ha scritto:
